packages/p2p/src/components/orders/order-table-row.jsx

```diff
@@ -9,10 +10,13 @@ import ServerTime from 'Utils/server-time';
 import { createExtendedOrderDetails } from 'Utils/orders';
 import { useStores } from 'Stores';
 
-const OrderRowComponent = observer(({ data: order, style }) => {
+const OrderRowComponent = observer(component_props => {
+    const { style, is_active, measure } = component_props;
```

Comment:
Can we make these computed and use them directly from the store?

Reply:
No, we could not (for now). because currently, we have to use different types of components for rendering rows(DataList and InfiniteLoaderList) by merging Aaron card this problem will solve and we can use a. component for both